Documentation/Signoffs/jiegwuatu42-web_wireless-signoff.md

```diff
@@ -7,22 +7,36 @@ The Wireless acquisition of data helps effortlessly boost the quality of life. T
 
 The subsystem constraints are placed to ensure that it can fulfill its job in a satisfactory manner. Constraints on the subsystem are as follows:
 1. The server host machine must possess adequate storage space for the server code to store all collected data.
-2. The data will be encrypted with an undisclosed algorithm
+2. The data will be encrypted and decrypted with an undisclosed algorithm using AES Symmetric Encryption.  
+3. The server must be able to be established and connected to the internet.
+4. The server must have adequate storage for user data.
+5. The data received must be in legible format.
 
 
 ## BUILDABLE SCHEMATIC OF SUBSYSTEM
 
+<img width="350" alt="4G Connection" src="https://user-images.githubusercontent.com/110966922/219462337-5e72409b-17d1-4c59-8d68-eab86c8fda1f.png">
+<img width="432" alt="Wifi Connection" src="https://user-images.githubusercontent.com/110966922/219462340-b16763c0-cbe4-4803-ba0f-be77fbb212e4.png">
 
 
+Server will be made using HTML. It will receive info from TCP Connection provided from UART. 
 
 ## ANALYSIS OF CHOSEN COMPONENT(S)
 
-
+The team has chosen HTML5 UP to statisfy constraints and jobs of this system for the server/UI:
```

Comment:
No components have been specified. Once they are specified this section needs to show how the components satisfy the constraints.

Reply:
The buildable schematic should show only this system, not the entire project.

Reply:
HTML is "the language for describing the structure of Web pages." It is only one part of a web server. The other parts need to be specified.

Update...The next section mentions cherokee, which addresses the above comment. Cherokee needs to be mentioned in this section.